app/Repositories/HistoryRepository.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Repositories;
+
+class HistoryRepository extends Repository
+{
+    public function create($date, $argument1, $argument2, $command, $result)
+    {
+        $content = $this->all();
+
+        $content[] = [
+            'date' => $date,
+            'first_operand' => $argument1,
+            'second_operand' => $argument2,
+            'sign' => $command,
+            'result' => $result
+        ];
+
+        file_put_contents(prepare_file_path("{$this->dirName}/{$this->fileName}"), json_encode($content));
+    }
```

Comment:
let's move `create` logic to the base class too

```suggestion
    {
        return parent::create([
            'date' => $date,
            'first_operand' => $argument1,
            'second_operand' => $argument2,
            'sign' => $command,
            'result' => $result
        ]);
    }
```